src/components/Item.js

```diff
@@ -20,17 +19,43 @@ const Item = ({ item, canBeSaved }) => {
     <Box
       display="flex"
       flexDirection="column"
-      direction="column"
       pt={3.5}
       px={2}
+      alignItems="center"
     >
-      <Typography variant="h1" align="center">{item.title}</Typography>
+      <Box
+        display="flex"
+        flexDirection="column"
+        pb={2}
+        alignItems="center"
+      >
+        <Typography variant="h1" align="center">{item.title}</Typography>
 
-      {canBeSaved ? <SaveButton resourceSlug={item.slug} /> : null}
+        {canBeSaved ? <SaveButton resourceSlug={item.slug} /> : null}
+      </Box>
 
-      {item.type === 'external_link'
-        ? <LinkUi href={item.link} target="_blank" color="inherit">Go to resource</LinkUi>
-        : richTextHelper(item.content, (node) => richTextTransformer(node))}
+      { (() => {
+        switch (item.type) {
+          case 'external_link':
+            return (
+              <Button
+                variant="outlined"
+                disableElevation
+                size="small"
+                onClick={() => {
+                  window.open(item.link, '_newtab');
```

Comment:
Could you add the `noopener` flag? ([ref](https://developer.mozilla.org/en-US/docs/Web/API/Window/open#noopener))

Reply:
Done in another PR

Reply:
If I'm not mistaken… you did add it in the other PR, but only on the leave site thing? Not on the external link?